<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTPlayer</class>
 <widget class="QWidget" name="QTPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#openButton:!hover
{
    background-color: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(120, 120, 120, 255),stop:0.495      rgba(39, 39, 39, 255),stop:0.505 rgba(39, 39, 39, 255),stop:1 rgba(120, 120, 120, 255));
    border:none;
    border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;黑体&quot;;
}
#openButton:hover
{
    background-color: qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(39, 39, 39, 255),stop:0.495      rgba(120, 120, 120, 255),stop:0.505 rgba(120, 120, 120, 255),stop:1 rgba(39, 39, 39, 255));
    border:none;
    border-radius:5px;
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;黑体&quot;;
}
#playButton{background-color: rgba(255, 255, 255,0);}

/*#playButton:hover{image: url(:/QTPlayer/Resources/PlayN.png);}

#playButton:!hover{image: url(:/QTPlayer/Resources/PlayH.png);}

#playButton:pressed{image: url(:/QTPlayer/Resources/PlayP.png);}
*/
 

#bottemWidget{
background-color: rgba(0, 0, 0, 122);
}

#totalHour{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#totalMinute{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#totalSecond{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#currentHour{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#currentMinute{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}
#currentSecond{
    color: rgb(255, 255, 255);
	font: 75 8pt &quot;黑体&quot;;
}</string>
  </property>
  <widget class="QWidget" name="bottemWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>800</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QSlider" name="playslider">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>601</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QWidget" name="currentContainer" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="currentLayout">
      <item>
       <widget class="QLabel" name="currentHour">
        <property name="text">
         <string>0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentMinute">
        <property name="text">
         <string>0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentSecond">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="totalContainer" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="totalLayout">
      <item>
       <widget class="QLabel" name="totalHour">
        <property name="text">
         <string>0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalMinute">
        <property name="text">
         <string>0:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalSecond">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="volume_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/volume);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>22</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>22</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="current_time">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>800</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>60</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/hn);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>5</y>
     <width>60</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/hk);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>5</y>
     <width>60</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/1);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>460</y>
     <width>16</width>
     <height>100</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>bottemWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>verticalSlider</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>QTPlayer</receiver>
   <slot>openVideoFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>-22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderPressed()</signal>
   <receiver>QTPlayer</receiver>
   <slot>sliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playslider</sender>
   <signal>sliderReleased()</signal>
   <receiver>QTPlayer</receiver>
   <slot>sliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>QTPlayer</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openVideoFile()</slot>
  <slot>sliderPress()</slot>
  <slot>sliderRelease()</slot>
  <slot>setPos(int)</slot>
  <slot>play()</slot>
  <slot>setVolume(int)</slot>
  <slot>mirrorLeftAndRight()</slot>
  <slot>mirrorUpAndDown()</slot>
 </slots>
</ui>
